.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Title
+
+Please write a descriptive title that describes
+what you've fixed (or created) with this Pull Request.
+
+# Summary
+
+Fixes #Issue_Number
+
+>note: The "Fixes #nnn" syntax in the PR description causes
+>GitHub to close the issue when this PR is merged. See 
+> [this GitHub help document](https://help.github.com/articles/closing-issues-via-commit-messages/)
```

Comment:
since this will show up as-is in that box, I'd remove links. Sorry for asking you that yesterday without understanding how this works. :disappointed: